client/components/telecom/telephony/group/number/feature/ovhPabx/menu/edit/telephony-group-number-feature-ovh-pabx-menu-edit.html

```diff
@@ -294,11 +290,11 @@ <h2 data-ng-if="['greetSound', 'invalidSound', 'greetSoundTts', 'invalidSoundTts
                         <em data-translate="telephony_number_feature_ovh_pabx_step_rule_no_tts_added"></em>
                     </div>
 
-                    <label class="section-item item-choice full-width pointer dtable table-fixed"
+                    <label class="section-item item-choice w-100 dtable table-fixed"
                            data-ng-if="$ctrl.menuCtrl.ovhPabx.tts.length > 0"
                            data-ng-repeat="tts in $ctrl.menuCtrl.ovhPabx.tts | orderBy:'text' track by $index">
                         <div class="dtr">
-                            <div class="dtd vertical-middle radio-content">
+                            <div class="d-table-cell valign-middle radio-content">
                                 <flat-radio>
```

Comment:
We have to remove `<flat-radio>` directive.

Reply:
in a future time